layouts/partials/contact-identifier.html

```diff
@@ -24,7 +24,8 @@
         <div class="tablet:grid-col-auto twitter-container">
           <a href="{{ $translation.footer.twitter_link }}" class="twitter-link">
             <span>{{ replace $translation.footer.twitter_text "%" "" }}</span>
-            <svg class="usa-icon" aria-hidden="true" focusable="false" role="img" height="28" width="28">
+            <svg class="usa-icon" aria-hidden="true" focusable="false" height="28" width="28" role="graphics-symbol" aria-label="twitter icon">
+              <title>twitter icon</title>
```

Comment:
I think we are going to need aria-label and title translated here. You can create a new ticket fo this.